src/MultiversX/GasEstimator.h

```diff
@@ -6,19 +6,20 @@
 
 #pragma once
 
-#include <string>
 #include "NetworkConfig.h"
+#include <string>
 
 namespace TW::MultiversX {
 
 class GasEstimator {
     NetworkConfig networkConfig;
+
 public:
     GasEstimator(const NetworkConfig& networkConfig);
 
-    uint64_t forEGLDTransfer(size_t dataLength) const;
-    uint64_t forESDTTransfer(size_t dataLength) const;
-    uint64_t forESDTNFTTransfer(size_t dataLength) const;
+    uint64_t forEGLDTransfer(size_t dataLength, bool isGuarded) const;
```

Comment:
maybe use different method instead of bool argument?

Reply:
Yes, will add separate methods :+1: